template/project.yml

```diff
@@ -15,21 +15,30 @@ project:
   # These are random strings used to configure the hash salt.
   hash_salt: ${project.hash_salt}
 
-# Configuration settings for new git repository.
-git:
-  default_branch: master
-  remotes:
-    # Defining git remotes allows builds deployed via CI.
-    - bolt8@svn-5223.devcloud.hosting.acquia.com:bolt8.git
-
 drush:
   aliases:
     # The remote environment from which the database will be pulled.
     remote: ${machine_name}.test
     # The local environment against which all local drush commands are run.
     local: self
 
-# Hosting environment flags.
-# Examples: acsf (Acquia Cloud Site Factory), ac (Acquia Cloud)
-# hosting: "acsf"
-# hosting: "ac"
+# Define deployment environments, and git repositories here here.
+# Defining git remotes allows builds deployed via CI.
+# Defining hosting allows certain artifacts to be deployed for different hosts
+
+deployments:
+  - acsf
+  - myhosting
+
+acsf:
+  hosting: acsf
+  gits:
+    - bolt8@svn-5223.devcloud.hosting.acquia.com:bolt8.git
+    - bolt@bolt.com:bolt8.git
+  name: Profserv ACSF
+  
+myhosting:
+  hosting: ace
```

Comment:
I would rename the default remote `default` and leave it uncommented. I would then comment out the ACSF part since that won't be very common.

You may also need to include, `default_branch: master` since that was included in the other section.